### PR TITLE
Added AppData manifest file for Gnome Software.

### PR DESCRIPTION
**Pull request by xvitaly**
Added AppData manifest file for Gnome Software.

### CODE REVIEW

File: resources/nheko.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <id>nheko.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv3+ and Public Domain and MIT and Boost and Apache License and CC-BY-4.0</project_license>
+  <name>nheko</name>
+  <summary>Desktop client for the Matrix protocol</summary>
+  <description>
+    <p>
+      The motivation behind the project is to provide a native desktop app
+      for Matrix that feels more like a mainstream chat app (Riot,
+      Telegram etc) and less like an IRC client.
```
Comment (MTRNord) [inline resources/nheko.appdata.xml]: Feels/Sounds like matrix/nheko is a IRC protocol which is isn't. Maybe leave the IRC part out as neither nheko nor Matrix directly has something to do with IRC (and yes it has a bridge and yes both are chat programs)

File: resources/nheko.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Vitaly Zaitsev <vitaly@easycoding.org> -->
+<component type="desktop">
+  <id>nheko.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv3+ and Public Domain and MIT and Boost and Apache License and CC-BY-4.0</project_license>
```
Comment (MTRNord) [inline resources/nheko.appdata.xml]: Not sure but shouldn't here be just the license of nheko and not of all of its dependencies?

Comment (xvitaly) [reply resources/nheko.appdata.xml]: License of main project and all it's dependencies.

File: resources/nheko.appdata.xml (hunk 1 above)
Comment (xvitaly) [reply resources/nheko.appdata.xml]: I got this information from Readme.md.

Comment (MTRNord) [reply resources/nheko.appdata.xml]: Hm ok :) Still leaving it here maybe for mujx to read it (didn't compare to the nheko README.)

Comment (mujx) [reply resources/nheko.appdata.xml]: I agree with @MTRNord. Could you use the description from https://github.com/mujx/nheko/blob/master/.ci/linux/create-packages.sh#L55

Comment (xvitaly) [reply resources/nheko.appdata.xml]: @mujx Description need to be longer than one row.

Comment (mujx) [reply resources/nheko.appdata.xml]: Then I guess you can stop at `chat app`.

Comment (xvitaly) [reply resources/nheko.appdata.xml]: @mujx Fixed.

**Pull request merged**